posthog-node/src/feature-flags.ts

```diff
@@ -396,11 +397,19 @@ class FeatureFlagsPoller {
       clearTimeout(this.poller)
       this.poller = undefined
     }
-
     this.poller = setTimeout(() => this._loadFeatureFlags(), this.getPollingInterval())
```

Comment:
We might need to tweak this too, right? We could probably check for the `inflightRequest` **above** this